docs/release/release.md

```diff
@@ -0,0 +1,34 @@
+## Preparing the release
+
+1. Create a release branch in the form of `release/[version]`
+2. Freeze the versions of each component on the requirements.yaml file to this branch.
```

Comment:
Can you add some instructions on how to get the latest versions here?
Like we do on circleci

Reply:
Done